src/fhirConfig.ts

```diff
@@ -113,9 +113,19 @@ export interface Profile {
     resources?: Resources;
 }
 
```

Comment:
Add a comment here explaining ProductInfo is used in generating the CapabilityStatement
```suggestion
/**
   * Used in the generation of the CapabilityStatement
   */
```